data/json/recipes/other/materials.json

```diff
@@ -239,6 +318,45 @@
       [ [ "material_sand", 25 ] ]
     ]
   },
+  {
+    "result": "10l_zinc",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "autolearn": true,
+    "using": [ [ "forging_standard", 10 ], [ "bronzesmithing_tools", 1 ] ],
+    "tools": [ [ [ "casting_mold", -1 ] ] ],
+    "components": [ [ [ "5l_zinc", 2 ] ] ]
+  },
+  {
+    "result": "5l_zinc",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "autolearn": true,
+    "using": [ [ "forging_standard", 5 ], [ "bronzesmithing_tools", 1 ] ],
+    "tools": [ [ [ "casting_mold", -1 ] ] ],
+    "components": [ [ [ "1l_zinc", 5 ] ] ]
+  },
+  {
+    "result": "1l_zinc",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "autolearn": true,
+    "using": [ [ "forging_standard", 1 ], [ "bronzesmithing_tools", 1 ] ],
+    "tools": [ [ [ "casting_mold", -1 ] ] ],
+    "components": [ [ [ "scrap_aluminum", 3780 ] ] ]
```

Comment:
Why are you making zinc out of aluminum? We don't have scrap zinc for you to make this recipe with. Unless you want to go with the atomic weight item... which given the amount I'm assuming was the intention.

Reply:
That was a copy/paste failure. It should read "zinc_metal", I hope that's OK?

Reply:
In this case you'd be looking at 3978. Not sure how you got this number.